docs/code-standards.md

```diff
@@ -0,0 +1,35 @@
+# Code & Documentation Standards
+
+This document details our goals for code quality, coverage and documentation standards. At the time
+of this writing, these are standards that all of our code may not meet. We should always try to 
+improve our code to meet them.
```

Comment:
'our code' is not exactly right since there's also the documentation to consider.
maybe: our code and documentation